app.py

```diff
@@ -59,6 +60,10 @@ def __init__(
 
         self.checked_repos: Set[str] = set()
 
+        deployment_name = self.oc.infra_namespace  # mi runs in infra (submit_mi() in workflows.py in common)
```

Comment:
deployment_name variable is usually `ocp4-stage`, `zero-prod` across thoth components, I think we should be consistent on stored data with `s3://{bucket_name}/{bucket_prefix}/{deployment_name}/{application_specific_path} 
`, wdyt?

Reply:
this should be consistent with it
right now for mi-scheduler it is `s3://thoth/data/thoth-test-core/app_specific_path/`

Reply:
or did you mean something else?

Reply:
So `Path(f"{deployment_name}/{subdir}/thoth-sli-metrics/kebechet-update-manager/")` is `app_specific_path`? In that case `deployment_name` would be `namespace` variable I would say

Reply:
well but as you've said
> deployment_name variable is usually `ocp4-stage`, `zero-prod` across thoth components
doesn't it mean that deployment_name variable is namespace?

Reply:
> So `Path(f"{deployment_name}/{subdir}/thoth-sli-metrics/kebechet-update-manager/")` is `app_specific_path`?

no, `app_specific` path in case is only `{subdir}/thoth-sli-metrics/kebechet-update-manager/`

Reply:
changed so that `deployment_name` is used instead, I think that's resolved for now :)